tools/cgdiff/include/NodeSummary.h

```diff
@@ -0,0 +1,63 @@
+/**
+ * File: NodeSummary.h
+ * License: Part of the MetaCG project. Licensed under BSD 3 clause license. See LICENSE.txt file at
+ * https://github.com/tudasc/metacg/LICENSE.txt
+ */
+#pragma once
+#include <Callgraph.h>
+#include <string>
+
+/**
+ * Summary of a node in a call graph for comparison purposes.
+ */
+struct NodeSummary {
+  std::string name;
+  bool hasBody;
+  std::unordered_set<std::string> callees;  // use function names as unique identifier
```

Comment:
```suggestion
  // use function names as unique identifier
  std::unordered_set<std::string> callees;
```
Please here and elsewhere.